src/en/users-creating.md

```diff
@@ -20,10 +20,10 @@ controller.
 At this stage, the user is registered to a controller and is allowed to perform
 the following additional actions (in the context of the controller):
 
-- List controllers (`juju controllers`)
-- Show controller details (`juju show-controller`)
-- List enabled controller users (`juju users`)
-- Show details for controller users (`juju show-user`)
+ - List controllers (`juju controllers`)
+ - Show controller details (`juju show-controller`)
+ - List enabled controller users (`juju users`)
+ - Show details for controller users (`juju show-user`)
```

Comment:
Does adding an extra space do something special?

Reply:
it is more proper, depending on which of the many versions of markdown syntax you subscribe too. It doesn't make any significant difference to us, except maybe it looks nicer?